pkg/kubelet/eviction/helpers_test.go

```diff
@@ -566,6 +570,242 @@ func TestParseThresholdConfig(t *testing.T) {
+				},
+				{
+					Signal:   evictionapi.SignalNodeFsInodesFree,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.05,
+					},
+				},
+			},
+		},
+		"disk eviction values in percentages": {
+			statements: map[string]string{"imagefs.available": "15%", "nodefs.available": "10.5%"},
+			expectErr:  false,
+			expectThresholds: []evictionapi.Threshold{
+				{
+					Signal:   evictionapi.SignalMemoryAvailable,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Quantity: quantityMustParse("100Mi"),
+					},
+				},
+				{
+					Signal:   evictionapi.SignalNodeFsAvailable,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.105,
+					},
+				},
+				{
+					Signal:   evictionapi.SignalImageFsAvailable,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.15,
+					},
+				},
+				{
+					Signal:   evictionapi.SignalImageFsInodesFree,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.05,
+					},
+				},
+				{
+					Signal:   evictionapi.SignalNodeFsInodesFree,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.05,
+					},
+				},
+			},
+		},
+		"inode eviction values": {
+			statements: map[string]string{"imagefs.inodesFree": "150Mi", "nodefs.inodesFree": "100Mi"},
+			expectErr:  false,
+			expectThresholds: []evictionapi.Threshold{
+				{
+					Signal:   evictionapi.SignalMemoryAvailable,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Quantity: quantityMustParse("100Mi"),
+					},
+				},
+				{
+					Signal:   evictionapi.SignalNodeFsAvailable,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.1,
+					},
+				},
+				{
+					Signal:   evictionapi.SignalImageFsAvailable,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Percentage: 0.15,
+					},
+				},
+				{
+					Signal:   evictionapi.SignalImageFsInodesFree,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Quantity: quantityMustParse("150Mi"),
+					},
+				},
+				{
+					Signal:   evictionapi.SignalNodeFsInodesFree,
+					Operator: evictionapi.OpLessThan,
+					Value: evictionapi.ThresholdValue{
+						Quantity: quantityMustParse("100Mi"),
+					},
+				},
+			},
+		},
+	}
+	for testName, testCase := range testCases {
+		thresholds, err := parseThresholdStatements(testCase.statements)
+		if testCase.expectErr != (err != nil) {
+			t.Errorf("Err not as expected, test: %v, error expected: %v, actual: %v", testName, testCase.expectErr, err)
+		}
+		var errcheck bool
+		for _, a := range thresholds {
+			for _, b := range testCase.expectThresholds {
+				if a.Signal == b.Signal {
+					errcheck = compareThresholdValue(a.Value, b.Value)
```

Comment:
```suggestion
					errcheck := compareThresholdValue(a.Value, b.Value)
```

No reason to declare it above.

Reply:
Ok @kannon92, I will update the same.